src/main/java/org/folio/circulation/support/ItemRepository.java

```diff
@@ -363,4 +363,8 @@ private ItemRepository(
       .thenComposeAsync(this::fetchMaterialType)
       .thenComposeAsync(this::fetchLoanType);
   }
+
+  public static ItemRepository fetchItemOnlyInstance(Clients clients) {
```

Comment:
How about renaming this method? or creating the constructor with default values? It seems to me the name

> fetchItemOnlyInstance 

means that we get some an item information

Reply:
Maybe `itemOnlyItemRepository`? Any other ideas?

Reply:
What does this method do, fetch only the item, without the holdings or instance records? Or without other related records e.g. location?

Why is it useful not to fetch those other records in this situation?

Reply:
This method creates an instance of `ItemRepository` that won't fetch `locations`, `material types` and `loan type` for items when call  (instance and holding will be fetched).

> Why is it useful not to fetch those other records in this situation?

Because they are not required for the flow, it is just extra calls that will affect performance.

Reply:
> This method creates an instance of `ItemRepository` that won't fetch `locations`, `material types` and `loan type` for items when call (instance and holding will be fetched).

Hmm, ok, yeah, I don't know of a good name to describe that. The booleans were a compromise that likely indicate the need for two separate approaches to fetching item records.

> > Why is it useful not to fetch those other records in this situation?
> 
> Because they are not required for the flow, it is just extra calls that will affect performance.

Ok. I think there is a trade off between improving performance and adding complexity to the system.

Reply:
Ok, will rename to `noLocationMaterialTypeAndLoanTypeInstance`. Let me know if there is a better name for the method.